site/rest.md

```diff
@@ -18,13 +18,41 @@ curl ... -H X-RequestedBy:MyClient ... -X POST .../scaling
 If you do not pass in the X-Requested-By header, then you'll get a 400 (bad request) response without any details explaining why the request is bad.
 The X-Requested-By header is not needed for requests that only read, for example when you GET any of the Operator's REST endpoints.
 
-If using `curl`, you can use the `-k` option to bypass the check to verify that the operator's certificate is trusted (instead of `curl --cacert`).
+Before using the sample script below, you must:
 
-Here is a small sample BASH script that may help to prepare the necessary token, certificates, and such, to call the operator's REST services:
+* Update it to ensure it has the correct service account, namespaces, etc., and it points to the `values.yaml`
+  that you used to install the operator (so that it can get the certificates),
+* Add your operator's certificate to your operating system's trust store (see below), and
+* If you are using a self-signed certificate and your client is macOS, you may need to update the version of `curl`
+  you have installed.  The version of curl that ships with macOS High Sierra (`curl 7.54.0 (x86_64-apple-darwin17.0)
```

Comment:
The version of curl -> The version of CURL

Reply:
Do you really need to add the cert to your os's trust store once you've installed the darwin version of CURL? i.e. I've been able to do curl --cacert on my hosted linux box without adding the cert to my trust store.